src/test/java/com/endava/soa_5/tools/PropertyReaderTest.java

```diff
@@ -0,0 +1,21 @@
+package com.endava.soa_5.tools;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+class PropertyReaderTest {
+    PropertyReader propertyReader;
+
+    @BeforeEach
+    void setUp() {
+        propertyReader = new PropertyReader("Configuration.properties");
+    }
+
+    @Test
```

Comment:
If decided cover functionality with tests, a negative one can be added

Reply:
Done!